src/map/utils/charutils.cpp

```diff
@@ -711,6 +711,8 @@ namespace charutils
                 PChar->clearCharVarsWithPrefix("jugpet-");
             }
         }
+        const auto [_, accid] = charutils::getCharIdAndAccountIdFromName(PChar->getName());
```

Comment:
this function (LoadChar) already queries from the chars table
https://github.com/LandSandBoat/server/blob/3ae70de4ae2263a43b8b141d210697eeade1638a/src/map/utils/charutils.cpp#L395

you can add `accid` to there and then set it like this

```
PChar->accid = rset->get<uint32>("accid");
```